### ISSUE TITLE
Asterisk being moved into @keyframes block.

### ISSUE THREAD

**Issue by jnorth** [postcss/autoprefixer#399]
Not sure of a better way to describe this:

```css
.portfolio {
  *zoom: 1;
}

@keyframes portfolio-fadein {
  0%   { opacity: 0; }
  100% { opacity: 1; }
}
```

compiled with autoprefixer 5.0.0 (using gulp-autoprefixer 2.1.0):

```css
.portfolio {
  *zoom: 1; }

@-webkit-keyframes portfolio-fadein {

  *0% {
  *  *opacity: 0; }

  *100% {
  *  *opacity: 1; } }

@keyframes portfolio-fadein {
  0% {
    opacity: 0; }

  100% {
    opacity: 1; } }

```

Can't reproduce on codepen: http://codepen.io/anon/pen/vEZmWw

**Comment by jnorth** [comment]
Tried with the CLI tool:

```
$ autoprefixer -v
autoprefixer 5.0.0

$ cat test.css 
.portfolio {
  *zoom: 1;
}

@keyframes portfolio-fadein {
  0%   { opacity: 0; }
  100% { opacity: 1; }
}

$ autoprefixer test.css 

$ cat test.css 
.portfolio {
  *zoom: 1;
}

@-webkit-keyframes portfolio-fadein {

  *0% {
  *  *opacity: 0;
  *}

  *100% {
  *  *opacity: 1;
  *}
}

@keyframes portfolio-fadein {
  0%   { opacity: 0; }
  100% { opacity: 1; }
}
```

it works as expected with `4.0.0`:

```
$ autoprefixer -v
autoprefixer 4.0.0

$ cat test.css 
.portfolio {
  *zoom: 1;
}

@keyframes portfolio-fadein {
  0%   { opacity: 0; }
  100% { opacity: 1; }
}

$ autoprefixer test.css 

$ cat test.css 
.portfolio {
  *zoom: 1;
}

@-webkit-keyframes portfolio-fadein {
  0%   { opacity: 0; }
  100% { opacity: 1; }
}

@keyframes portfolio-fadein {
  0%   { opacity: 0; }
  100% { opacity: 1; }
}
```

**Comment by ai** [comment]
Fuck, yeap in this rare case with only one decl, PostCSS will take `*`. I will try to find solution.

**Comment by ai** [comment]
Fixed https://github.com/postcss/postcss/commit/1bcbfb9e202582af5305405c8bcbc0ba64cec51c

**Comment by ai** [comment]
Released in PostCSS 4.0.2. Please update npm dependencies.
